provision/acc_provision/templates/calico-bgp-config.yaml

```diff
@@ -0,0 +1,12 @@
+#This section disables BGP full mesh between the K8s nodes and set the serviceClusterIPs and serviceExternalIPs subnets so that they can be advertised by eBGP. These subnets are the Service and External Service subnets in Kubernetes.
+apiVersion: projectcalico.org/v3
+kind: BGPConfiguration
+metadata:
+  name: default
+spec:
+  logSeverityScreen: Info
+  nodeToNodeMeshEnabled: false
+  serviceClusterIPs:
+  - cidr: {{ config.net_config.cluster_svc_subnet }}
+  serviceExternalIPs:
+  - cidr: {{ config.net_config.extern_dynamic }}
```

Comment:
Please include serviceLoadBalancerIPs here as discussed

Reply:
Added it.